core/src/main/scala/org/apache/spark/deploy/worker/Worker.scala

```diff
@@ -467,6 +467,15 @@ private[deploy] class Worker(
         }.foreach { dir =>
           logInfo(s"Removing directory: ${dir.getPath}")
           Utils.deleteRecursively(dir)
+
```

Comment:
unrelated to your change -- can you add 2 space indentation on line 466 `!Utils.doesDirectoryContainAnyNewFiles(dir, APP_DATA_RETENTION_SECONDS)` as its a continuation?